src/recombinations.jl

```diff
@@ -481,6 +481,21 @@ function POS(v1::T, v2::T; rng::AbstractRNG=Random.GLOBAL_RNG) where {T <: Abstr
     return c1,c2
 end
 
+# Combinatorial crossovers
+# ----------------------
+
+"""Subset crossover"""
+function SXO(v1::T, v2::T; rng::AbstractRNG=Random.GLOBAL_RNG) where {T <: AbstractVector}
+    l = length(v1) # get number of available elements
+    K = sum(v1) # get required subset size
+    pooled = findall(v1 .| v2) # pool parents selections
+    c1 = falses(l) # init child 1
+    c2 = falses(l) # init child 2
+    c1[shuffle(pooled)[1:K]] .= true # fill child 1 with sample from pool w/o replacement
```

Comment:
You need to add as a `shuffle` first parameter `rng` object.

Reply:
oops, good catch! it's fixed now.